package.json

```diff
@@ -16,7 +16,7 @@
     "@babel/compat-data": "^7.19.1",
     "@bcrs-shared-components/action-chip": "1.1.3",
     "@bcrs-shared-components/breadcrumb": "2.1.3",
-    "@bcrs-shared-components/business-lookup": "1.1.3",
+    "@bcrs-shared-components/business-lookup": "1.1.5",
```

Comment:
To pick up the shared component I recently updated ([PR 156](https://github.com/bcgov/bcrs-shared-components/pull/156)).